File: tests/openmp_examples/devices/target_data.6.c
#pragma omp target data if(N>1000000) map(from: p[0:N])
#pragma omp target if (N>1000000) map(to: v1[:N], v2[:N])
#pragma omp parallel for
#pragma omp target if (N>1000000) map(to: v1[:N], v2[:N])
#pragma omp parallel for

File: tests/openmp_examples/devices/target_data.6.f90
!$omp    target data if(N>THRESHOLD) map(from: p)
!$omp       target if(N>THRESHOLD) map(to: v1, v2)
!$omp          parallel do
!$omp       end target
!$omp       target if(N>THRESHOLD) map(to: v1, v2)
!$omp          parallel do
!$omp       end target
!$omp    end target data
